packages/file_selector/file_selector_android/android/src/main/java/io/flutter/plugins/file_selector/FileSelectorDelegate.java

```diff
@@ -4,18 +4,28 @@
 
 package io.flutter.plugins.file_selector;
 
+import android.Manifest;
```

Comment:
Review some of these imports since some are unused. You can check these using android studio or intellij "actions"